Yoast/wordpress-seo
# Stop showing twitter title and description when fallback on OG

## Summary


This PR can be summarized in the following changelog entry:

* N/A - It has not been released yet

## Relevant technical choices:

* When there is no twitter title set. The value of `twitter:title` fallback to the OG-title shouldn't be done. Instead of that do not show the meta tag for it and let Twitter rely on the present OG-title.
* The same applied for the twitter description.

## Test instructions

This PR can be tested by following these steps:
 
* Set a OG title and don't set the twitter title. Press save and verify no `twitter:title` metatag being rendered.
* Empty the OG title and the fallback will be the post title.
* Do the same for OG description and twitter description.

## UI changes

## Documentation

## Quality assurance


Fixes #
